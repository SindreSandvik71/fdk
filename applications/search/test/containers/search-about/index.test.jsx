import React from 'react';
import { shallow } from 'enzyme';
import About from '../../../src/containers/search-about';

test('should render About correctly', () => {
  const wrapper = shallow(<About />);
  expect(wrapper).toMatchSnapshot();
});
