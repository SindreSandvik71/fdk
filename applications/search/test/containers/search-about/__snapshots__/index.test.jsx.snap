// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render About correctly 1`] = `
<div
  className="container"
>
  <SideEffect(DocumentMeta)
    description="Felles Datakatalog er en oversikt over hvilke data de ulike offentlige virksomhetene har, hvordan de henger sammen og hva de betyr. Datakatalogen gjør det mulig for deg å søke i opplysningene hos offentlige virksomheter og bruke de til for eksempel analyseformål."
    title="Om Felles datakatalog"
  />
  <div
    className="row"
  >
    <div
      className="col-md-8 col-md-offset-2"
    >
      <h1
        className="fdk-margin-bottom"
      >
        Felles datakatalog
      </h1>
      <div
        className="fdk-margin-bottom"
      >
        <p
          className="fdk-ingress"
        >
          

        </p>
        <p
          className="fdk-ingress"
        >
          Felles Datakatalog er en oversikt over hvilke data de ulike offentlige virksomhetene har, hvordan de henger sammen og hva de betyr. Datakatalogen gjør det mulig for deg å søke i opplysningene hos offentlige virksomheter og bruke de til for eksempel analyseformål.
        </p>
      </div>
      <div
        className="fdk-textregular"
      >
        <p
          dangerouslySetInnerHTML={
            Object {
              "__html": "Regjeringen sier i Digital Agenda at brukerne skal levere informasjon til offentlig forvaltning kun en gang, fordi forvaltningen selv skal sørge for å dele nødvendig informasjon med hverandre. Felles datakatalog er første steget for å nå dette målet. Samarbeidsorganet Skate (Styring og koordinering av tjenester i e-forvaltning) har beskrevet ambisjonsnivået for informasjonsforvaltning i tre trinn, og Felles datakatalog skal i første trinn skaffe en felles dataoversikt og en felles forståelse av informasjon i form av begreper. Neste trinn har fokus på tilgjengeliggjøring av informasjon i en felles API-katalog. I tredje trinn ligger alt til rette for at gjenbruk av data er hovedregel og for prinsippet om kun én gang.<br><br><b>Informasjonsforvaltning</b><br> Digitaliseringsrundskrivet påpeker at alle offentlige virksomheter skal ha en oversikt over hvilke data de håndterer, hva dataene betyr, hva de kan brukes til, hvilke prosesser de er en del av og hvem som kan bruke dataene (informasjonsforvaltning). Direktoratet for Forvaltning og IKT (Difi) utarbeider et felles rammeverk med standardiserte metoder og prinsipper som vil gjøre det enklere for virksomhetene som skal i gang med begrepsarbeid.",
            }
          }
        />
        <p
          dangerouslySetInnerHTML={
            Object {
              "__html": "<b>Import fra data.norge.no, geonorge.no m.fl.</b><br>Felles datakatalog importerer data-beskrivelser fra andre kataloger, blant annet <a href='https://data.norge.no' target='blank'>data.norge.no</a> (<a href='https://data.norge.no/data/direktoratet-forvaltning-og-ikt/datanorgeno-nasjonal-katalog-over-åpne-offentlige-datasett' target='blank'>API</a>) og geonorge.no. Dette gjøres ved å bruke DCAT-AP-NO som dataformat for å utveksle data-beskrivelser.",
            }
          }
        />
        <p>
          <b>
            Registrere datasett
          </b>
          <br />
          <Link
            replace={false}
            to="/about-registration"
          >
            Hjelp til å komme i gang med å registrere datasett
          </Link>
        </p>
        <p>
          <a
            href="https://registrering-fdk.ppe.brreg.no/"
            rel="noopener noreferrer"
            target="_blank"
            title="Lenke til registreringsløsning"
          >
            <span
              className="fdk-button fdk-button-default"
            >
              Kom i gang med registreringen
            </span>
          </a>
        </p>
      </div>
    </div>
  </div>
</div>
`;
